content/en/about/governance.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Governance and Decision Making"
+draft: false
+---
+
+# Abstract
+
+Scientific Python is a consensus-based community project.
```

Comment:
```suggestion
Scientific Python is a consensus-based community ecosystem.
```

Just a thought on wording, maybe project is better - it would require more rewording below.

Reply:
I've been trying to use project for the work going on in this GH org and ecosystem to refer to the loosely-federated collection of libraries.